src/image/p5.Image.js

```diff
@@ -681,92 +877,228 @@ p5.Image = class {
   }
 
   /**
-   * Applies an image filter to a <a href="#/p5.Image">p5.Image</a>
+   * Applies an image filter to the <a href="#/p5.Image">p5.Image</a> object.
+   * The preset options are:
    *
-   * THRESHOLD
-   * Converts the image to black and white pixels depending if they are above or
-   * below the threshold defined by the level parameter. The parameter must be
-   * between 0.0 (black) and 1.0 (white). If no level is specified, 0.5 is used.
+   * `INVERT`
+   * Inverts the colors in the image. No parameter is used.
    *
-   * GRAY
-   * Converts any colors in the image to grayscale equivalents. No parameter
-   * is used.
+   * `GRAY`
+   * Converts the image to grayscale. No parameter is used.
    *
-   * OPAQUE
-   * Sets the alpha channel to entirely opaque. No parameter is used.
+   * `THRESHOLD`
+   * Converts the image to black and white. Pixels with a grayscale value
+   * above a given threshold are converted to white. The rest are converted to
+   * black. The threshold must be between 0.0 (black) and 1.0 (white). If no
+   * value is specified, 0.5 is used.
    *
-   * INVERT
-   * Sets each pixel to its inverse value. No parameter is used.
+   * `OPAQUE`
+   * Sets the alpha channel to entirely opaque. No parameter is used.
```

Comment:
Super tiny, but this might read slightly more clearly if it's "to **be** entirely opaque"?

Reply:
Good suggestion–fixed!